commit be258fda95c5a89305aa7a1bf2530d307a293115
Author: Andrew Cagney <cagney@gnu.org>
Date:   Fri Sep 6 12:24:43 2019 -0400

    disable arch32 tests by default
    
    frysk-0.4-disable-arch32-tests.patch

diff --git a/frysk-common/m4/frysk_do_arch32_test.m4 b/frysk-common/m4/frysk_do_arch32_test.m4
index 6f8c2a13d..d8b3b7b54 100644
--- a/frysk-common/m4/frysk_do_arch32_test.m4
+++ b/frysk-common/m4/frysk_do_arch32_test.m4
@@ -39,13 +39,9 @@
 AC_DEFUN([FRYSK_DO_ARCH32_TEST], [
 AC_REQUIRE([AC_CANONICAL_TARGET])
 AC_ARG_ENABLE([arch32_tests],
-    [AS_HELP_STRING([--disable-arch32-tests],
-	[Disable the multiarch testing on 64-bit platforms.])],
+    [AS_HELP_STRING([--enable-arch32-tests],
+	[Enable the multiarch testing on 64-bit platforms.])],
     [],
-    [dnl The default:
-    case $target_cpu in #(
-    x86_64|powerpc64|ppc64) enable_arch32_tests=yes;; #(
-    *) enable_arch32_tests=no;;
-    esac])
+    [enable_arch32_tests=no])
 AM_CONDITIONAL([DO_ARCH32_TEST], [test "$enable_arch32_tests" = yes])
 ])
